packages/documentation/pages/usage/components/field-inline-edit.vue

```diff
@@ -0,0 +1,189 @@
+					</div>
+
+					<div>
+						<h4>Texts</h4>
+						<KtFieldText formKey="label" isOptional label="label" />
+						<KtFieldText
+							formKey="helpDescription"
+							isOptional
+							label="helpDescription"
+						/>
+						<div class="field-row">
+							<KtFieldText formKey="helpText" isOptional label="helpText" />
+						</div>
+					</div>
+				</div>
+			</KtForm>
+		</div>
+	</div>
+</template>
+
+<script lang="ts">
+import {
+	Kotti,
+	KtFieldInlineEdit,
+	KtForm,
+	KtFieldSingleSelect,
+} from '@3yourmind/kotti-ui'
+import { defineComponent, ref, computed, Ref } from '@vue/composition-api'
+
+import ComponentInfo from '~/components/ComponentInfo.vue'
+
+export default defineComponent({
+	name: 'DocumentationPageUsageComponentsFieldInlineEdit',
+	components: {
+		ComponentInfo,
+		KtFieldInlineEdit,
+		KtForm,
+		KtFieldSingleSelect,
+	},
+	setup() {
+		const isEditing = ref(false)
+		const values = ref<{ fieldValue: Kotti.FieldInlineEdit.Value }>({
+			fieldValue: null,
+		})
+
+		const settings = ref<{
+			helpDescription: Kotti.FieldText.Value
+			helpText: Kotti.FieldText.Value
+			booleanFlags: {
+				hideValidation: Kotti.FieldToggle.Value
+				isDisabled: Kotti.FieldToggle.Value
+				isLoading: Kotti.FieldToggle.Value
+				isMultiline: Kotti.FieldToggle.Value
+				isOptional: Kotti.FieldToggle.Value
+			}
+			label: Kotti.FieldText.Value
+			placeholder: Kotti.FieldText.Value
+			preventConfirmationOn: Kotti.FieldInlineEdit.ConfirmationValidation
+			tabIndex: Kotti.FieldNumber.Value
+			textStyle: Kotti.FieldInlineEdit.TextStyle | null
+			validation: Kotti.Field.Validation.Result['type']
+		}>({
+			helpDescription: null,
+			helpText: null,
+			booleanFlags: {
+				hideValidation: false,
+				isDisabled: false,
+				isLoading: false,
+				isMultiline: true,
+				isOptional: true,
+			},
+			label: 'Label',
+			placeholder: null,
+			preventConfirmationOn: Kotti.FieldInlineEdit.ConfirmationValidation.NEVER,
+			tabIndex: null,
+			textStyle: null,
+			validation: 'empty',
+		})
+
+		return {
+			component: KtFieldInlineEdit,
+			ktFieldInlineEditProps: computed(() => ({
+				helpDescription: settings.value.helpDescription,
+				helpText: settings.value.helpText,
+				hideValidation: settings.value.booleanFlags.hideValidation,
+				isDisabled: settings.value.booleanFlags.isDisabled,
+				isLoading: settings.value.booleanFlags.isLoading,
+				isOptional: settings.value.booleanFlags.isOptional,
+				isMultiline: settings.value.booleanFlags.isMultiline,
+				label: settings.value.label,
+				placeholder: settings.value.placeholder,
+				preventConfirmationOn: settings.value.preventConfirmationOn,
+				tabIndex: settings.value.tabIndex,
+				textStyle: settings.value.textStyle,
+				validator: () => ({
+					type: settings.value.validation,
+					text: 'Some validation text',
+				}),
+			})),
+			isEditing,
+			onConfirm: (newVal: Kotti.FieldInlineEdit.Events.Confirm) => {
+				window.alert(`Confirmed: ${newVal}`)
```

Comment:
Is this alert intended?

Maybe you can do the same as in form-fields docs: keeping an external variable and showing it below the `KtFieldInlineEdit`, so the docs shows how it changes based on what it's typed in the field